tests/aksdeployments_test.go

```diff
@@ -0,0 +1,64 @@
+package aks_test
+
+import (
+	"fmt"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	v1 "k8s.io/api/apps/v1"
+)
+
+var _ = Describe("AKS-Deployment Tests : ", func() {
+	//Adding delay just to let flux deploy the manifests.
+	time.Sleep(30 * time.Second)
```

Comment:
I wonder whether this should be external to the tests, as someone running this on an already built cluster will be sleeping for 30 seconds for no reason